StRoot/StTofHitMaker/StTofHitMaker.cxx

```diff
@@ -146,8 +146,8 @@ Int_t StTofHitMaker::Make()
    LOG_INFO << " getting the tof collection " << tofCollection << endm;
    if (tofCollection) {
       if ( GetNextRaw() ) {
-         // u_int ddl[4][10000] ;	/// content of up to 4 fibers
-     	   // u_int ddl_words[4] ;	/// the count of words (32bit) for above
+         // uint32_t ddl[4][10000] ;	/// content of up to 4 fibers
```

Comment:
Guessing this is done with a refactoring tool... which explains why comments are left in place (which I'm fine with).  Gives a little more confidence that the types remain invariant under the refactoring.

Reply:
Sorry to disappoint, it's just a tab character. My refactoring tool is `find . -type f -exec sed -i {} -e 's/u_int/uint32_t/g' \;` and `git add -p`. However, `clang-rename` can handle this too, but would require some extra setup.